adapters/src/lib.rs

```diff
@@ -36,15 +36,15 @@ mod tests;
 
 /// Weight trader that accepts multiple assets as weight fee payment.
 ///
-/// It uses `WeightToFee` in combination with a `NativePriceOracle` to set the right price for weight.
+/// It uses `ConvertWeightToFee` in combination with a `NativePriceOracle` to set the right price for weight.
```

Comment:
I think this one is still `WeightToFee`, as this is the name of the trait. For example after this you mention `NativePriceOracle` which is also a trait.

small thing...

```suggestion
/// It uses `WeightToFee` in combination with a `NativePriceOracle` to set the right price for weight.
```